user_data/strategies/bbrsi.py

```diff
@@ -0,0 +1,128 @@
+# --- Do not remove these libs ---
+from pandas import DataFrame
+# --------------------------------
+
+import talib.abstract as ta
+import freqtrade.vendor.qtpylib.indicators as qtpylib
+import numpy  # noqa
+from wao.WAOStrategy import WAOStrategy
+
+
+class bbrsi(WAOStrategy):
+    # Minimal ROI designed for the strategy.
+    # This attribute will be overridden if the config file contains "minimal_roi"
+
+    def __init__(self, config: dict):
+        super().__init__(config)
+        self.controller.setup("Freq_bbrsi")
+
+    brain = "Freq_bbrsi"
```

Comment:
move brain on top of init - same comment as mentioned in 002
apply for all strategies